### ISSUE TITLE
Hi, how can i show high, low, open ,close values?

### ISSUE THREAD

**Issue by AshotStdev** [andredumas/techan.js#226]

**Comment by evbarnett** [comment]
Are you referring to [this](http://bl.ocks.org/andredumas/06ad3573c0053d0e1fc7) type of graph?

**Comment by icastillejogomez** [comment]
I think @AshotStdev need the typical box on candlestick's hover with that information. I need it too 😞

**Comment by icastillejogomez** [comment]
The only solution I founded to this trouble is to use crosshair mouse axis. http://bl.ocks.org/andredumas/045f550b72ad46301130

**Comment by thibautquentinjacob** [comment]
@icastillejogomez Found a way to display OHLC information corresponding to the cursor position. Let me know if it's what you're after and I'll share the code.

![screenflow](https://user-images.githubusercontent.com/1339919/46376398-32887d80-c696-11e8-8ddc-3e08ff096a65.gif)

**Comment by icastillejogomez** [comment]
@thibautquentinjacob yes!!! This is exactly i'm looking for. I was referring about a hover box on the candle but this solution is better for me.

I'm supposing that you have a hover event that call back you the current candle data.

**Comment by thibautquentinjacob** [comment]
@icastillejogomez I did it the following way:
- Created HTML elements to hold the values for the different metrics
- Used [d3.bisector](https://github.com/d3/d3-array#bisector) to get the data index corresponding to the mouseX position
- From then, I just had to update the HTML elements using the values 

[Here is a gist](https://gist.github.com/thibautquentinjacob/2b20f62b81c744ed5547c670658e800d)  to help you reproduce that feature

**Comment by icastillejogomez** [comment]
Thank you so much @thibautquentinjacob 😄 👍

**Comment by thibautquentinjacob** [comment]
You're welcome! Let me know how it works out 😄

**Comment by icastillejogomez** [comment]
Make sense!! Tomorrow I'll test it and let you know if works out!

**Comment by icastillejogomez** [comment]
Works fine!!! thank you so much @thibautquentinjacob

**Comment by icastillejogomez** [comment]
@thibautquentinjacob Can we talk more privately? I am working in a startup and we are considering offer you a freelance work with techan library (less than a week). We would be very happy to know that you can help us
